src/utils/command.js

```diff
@@ -91,7 +91,7 @@ class BaseCommand extends Command {
   }
 
   // Find and resolve the Netlify configuration
-  async getConfig(cwd, state, token, argv) {
```

Comment:
This is taken from the top-level scope, so it does not need to be passed as argument.